src/Sulu/Bundle/ContactBundle/Controller/AccountMediaController.php

```diff
@@ -49,6 +49,20 @@ public function postAction($id, Request $request)
     }
 
     /**
+     * Replaces the medias of the account.
+     *
+     * @param $id
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function putAction($id, Request $request)
```

Comment:
We decided to use `PATCH`, as described in this article: http://williamdurand.fr/2014/02/14/please-do-not-patch-like-an-idiot/